prime-router/src/main/kotlin/fhirengine/translation/hl7/schema/ConfigSchemaReader.kt

```diff
@@ -44,6 +44,29 @@ object ConfigSchemaReader : Logging {
         return mergedSchema
     }
 
+    fun fromFile(
+        schemaUri: URI,
+        schemaClass: Class<out ConfigSchema<out ConfigSchemaElement>>,
+    ): ConfigSchema<*> {
+        // Load a schema including any parent schemas.  Note that child schemas are loaded first and the parents last.
+//        val schemaList = when (schemaUri.scheme) {
+//            null -> fromRelative(schemaName, folder, schemaClass)
+//            else -> fromUri(schemaUri, schemaClass)
+//        }
+
+        val schemaList = when (schemaUri.scheme) {
+            "classpath" -> fromUri(schemaUri, schemaClass)
```

Comment:
I don't think this is right... this means we can only support cases where it's a classpath.  The goal of this is epic is to support many schemes?

Reply:
how many scheme we are going to support by this PR?

Reply:
The code already supports all the schemes